src/components/PublicHeader.tsx

```diff
@@ -75,7 +82,7 @@ const PublicHeader = ({ user, org, userMemberships } : PublicHeaderProps) : JSX.
                         </View>
                         <>
                             { canOrganize && (
-                                <NextLink href={ `/organize/${userMemberships[0].organization.id}` }>
+                                <NextLink href={ `/organize/${currentOrg}` }>
```

Comment:
So here you would refer to ${organizeUrl} instead of ${currentOrg}, which is also a more precise description of the variable in question